packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -231,13 +231,18 @@ class UsersProfileClass extends Component<UsersProfileProps,UsersProfileState> {
     const currentFilter = query.filter ||  "all"
     const ownPage = currentUser && currentUser._id === user._id
     const currentShowLowKarma = (parseInt(query.karmaThreshold) !== DEFAULT_LOW_KARMA_THRESHOLD)
+    
+    const username = Users.getDisplayName(user)
+
+    const userPostsTitle = username.slice(-1) === "s" ? `${username}' Posts` :`${username}'s Posts`
+    const userCommentsTitle = username.slice(-1) === "s" ? `${username}' Comments` :`${username}'s Comments`
```

Comment:
Could also be written `username.endsWith("s")` which is a bit more direct (this works though).